app/root.tsx

```diff
@@ -76,28 +69,12 @@ export const loader: LoaderFunction = async ({ request }: LoaderArgs) => {
 };
 
 export default function App() {
-  const { tekstData, søkerData, dekoratørFragmenter } =
-    useLoaderData<typeof loader>();
-  const [språk, settSpråk] = useState<ELocaleType>(ELocaleType.NB);
-  const [erSamtykkeBekreftet, settErSamtykkeBekreftet] = useState(false);
-  const [endringsmeldingMottattDato, settEndringsmeldingMottattDato] =
-    useState('');
+  const { dekoratørFragmenter } = useLoaderData<typeof loader>();
 
   return (
-    <Dokument språk={språk}>
+    <Dokument språk={ELocaleType.NB}>
```

Comment:
Er det riktig at språk hardkodes her?

Reply:
Ja her får vi et problem ja! 

Denne ble hardkodet dette er i `root.tsx` for her blir `<html>` taggen rendret.
Når man går til `ba` så bruker man `ba.tsx` som "ba sin root fil".  Der henter man språk fra context satt i `ba.tsx`. Men det endrer jo ikkje språket som blir satt i `<html>` taggen. Den ligger ytterst of blir ikkje tilgjenglig når vi lager et nytt context på "et lavere nivå". (Selv om språk blir endrer på selve siden)

Da er spørsmålet om man kan endre dette `<html lang="?">` i språkvelger komponentet? 🥸 For da går det fint at den er bokmål som standard.

Reply:
Kan du lage et kort for å se nærmere på dette?

Reply:
Nytt kort her: https://trello.com/c/70wF6AKO/143-uu-krav-spr%C3%A5k-m%C3%A5-bli-satt-html-taggen